CONTRIBUTING.md

```diff
@@ -27,13 +27,13 @@ Exception: If a URL in a comment is too long, it can go over the limit. This hap
 
 * Braces go in separate lines. ([Allman style](http://en.wikipedia.org/wiki/Indent_style#Allman_style).)
 * If only one statement is contained inside the block, omit braces unless...
-    * This is part of a if-(else if-)else structure. All brace styles in the same structure should match (i.e. either non or all of them omit braces).
+    * This is part of an if-(else if-)else structure. All brace styles in the same structure should match (i.e. either non or all of them omit braces).
 
-#### Stetements Inside `if`, `while`, etc.
+#### Stetements inside `if`, `while`, etc.
```

Comment:
It might be better to make title styles consistent. We’re using Title Case everywhere else, so this should too.